### ISSUE TITLE
ImportError when running soda from CLI

### ISSUE THREAD

**Issue by mando222** [sodadata/soda-sql#384]
**Describe the bug**
When running soda from the CLI I get the following traceback:

```
Traceback (most recent call last):
  File "/usr/lib/command-not-found", line 28, in <module>
    from CommandNotFound import CommandNotFound
  File "/usr/lib/python3/dist-packages/CommandNotFound/CommandNotFound.py", line 19, in <module>
    from CommandNotFound.db.db import SqliteDatabase
  File "/usr/lib/python3/dist-packages/CommandNotFound/db/db.py", line 5, in <module>
    import apt_pkg
ImportError: libapt-pkg.so.6.0: cannot open shared object file: No such file or directory
```

**To Reproduce**
1. sudo apt-get install g++ unixodbc-dev python3-dev libssl-dev libffi-dev
2. sudo apt-get install build-essential cargo
3. pip3 install soda-sql-postgresql
4. soda

**Context**
Running Vagrant+Virtualbox with the attached vagrantfile 
[Vagrantfile.zip](https://github.com/sodadata/soda-sql/files/6787318/Vagrantfile.zip)
 
**OS**: Ubuntu 18.04.5 LTS
**Python Version**: 3.8.11
**Soda SQL Version**: unable to get info due to error.  Should be current as of 07/08/21
**Warehouse Type**: PostgreSQL

**Comment by vijaykiran** [comment]
This could be because of improper Python installation, can you please check if you have multiple versions of Python? If so uninstall and reinstall compatible Python version (3.7 or 3.8 or 3.9) and try again?

**Comment by tombaeyens** [comment]
@vijaykiran Is a wrong Python version something we could detect as the CLI starts so that we can print an appropriate error message?

**Comment by vijaykiran** [comment]
The setup tools are already configured to prevent installing on unsupported python version. In this case, I think it is related to multiple python versions installed.. Maybe this will help: https://askubuntu.com/questions/766246/missing-package-commandnotfound

**Comment by mando222** [comment]
This is definitely a python issue.  That said multiple versions of python shouldn't cause something like this.  It is common to have more than one version on any given box.

**Comment by vijaykiran** [comment]
It's not a problem if you have multiple versions, I think it depends on where pip was installing the packages to. Did you try the solution linked above? Perhaps that should help in resolving Python setup. 

Let me know if it resolves the issue. I can also try to reproduce it tonight. 

Also, you don't need to install odbc and other packages, as they are only needed if you want to install SQLServer support. The steps in getting started in our docs should suffice.

**Comment by mando222** [comment]
Ok after tracing this all back it seems that the issue lies with python3.6 and pip3's relationship in the context of Ubuntu 18.  Long story short if upgrading everything so it uses the newer python3.8 and pip3 correctly some dependencies that Ubuntu 18 need get a bit fudged up.  I ended up upgrading to Ubuntu 20 and all the issues went away.  Not a great fix but good to know that 18 has some issues here.  I have attached a working [Vagrantfile here](https://github.com/sodadata/soda-sql/files/6794092/Vagrantfile.zip)
